doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -200,19 +200,19 @@ Bluetooth samples
 * :ref:`peripheral_hids_mouse` and :ref:`central_uart` samples now come with the :ref:`ble_rpc_host` child image configuration overlay.
   The overlay shows how to configure an application running a serialized Bluetooth Low Energy Host.
   These samples run out the box with the :ref:`ble_rpc` library.
+* Added :ref:`central_and_peripheral_hrs` sample.
 
 Bluetooth mesh samples
 ----------------------
 
 * Added:
 
-  * :ref:`bluetooth_ble_peripheral_lbs_coex` sample, demonstrating how to combine Bluetooth mesh and Bluetooth Low Energy features in a single application.
+  * :ref:`bluetooth_ble_peripheral_lbs_coex`, demonstrating how to combine Bluetooth mesh and Bluetooth Low Energy features in a single application.
   * Support for :ref:`zephyr:nrf21540dk_nrf52840`.
-  * :ref:`central_and_peripheral_hrs` sample.
 
 * Updated:
 
-  * Updated some samples with support for :ref:`zephyr:thingy53_nrf5340` in non-secure configuration.
+  * :ref:`bluetooth_mesh_light`, :ref:`bluetooth_mesh_light_lc`, :ref:`bluetooth_mesh_light_switch` and :ref:`bluetooth_mesh_sensor_server` with support for :ref:`zephyr:thingy53_nrf5340`.
   * Updated some samples to use DTS overlays instead of KConfig configuration files for setting up external flash memory.
```

Comment:
Should these be listed as well? Your choice.
I listed at least those samples that got Thingy:53 support above.

You can also remove Updated from the bullet point.

Reply:
Two of them has been updated, and two has been added. Then we should be specific in what has been done for each of them. light and light_switch has been updated with non-secure and light_lc and sensor server/client has been added.